tests/memory_threshold_check_test.py

```diff
@@ -0,0 +1,71 @@
+import os
+import sys
+import pytest
+from unittest import mock
+from .mock_tables import dbconnector
+from utilities_common.general import load_module_from_source
+
+test_path = os.path.dirname(os.path.abspath(__file__))
+modules_path = os.path.dirname(test_path)
+scripts_path = os.path.join(modules_path, 'scripts')
+sys.path.insert(0, scripts_path)
+
+memory_threshold_check_path = os.path.join(scripts_path, 'memory_threshold_check.py')
+memory_threshold_check = load_module_from_source('memory_threshold_check.py', memory_threshold_check_path)
+
+@pytest.fixture()
+def case_1():
```

Comment:
what is cast_1? Could you please use a meaningful name?

Reply:
Renamed fixtures